code/game/gamemodes/extended.dm

```diff
@@ -4,17 +4,15 @@
 	required_players = 0
 
 
-/datum/game_mode/announce()
+/datum/game_mode/extended/announce()
```

Comment:
Did you make sure that there was a game_mode/announce() declared elsewhere to ensure there is a debug message encase people mess up with their new mode?